README.md

```diff
@@ -540,6 +541,104 @@ ActionDispatch::Http::Parameters::DEFAULT_PARSERS[Mime::Type.lookup(JSONAPI::MIM
 end
 ```
 
+## Sinatra example
+
+Using [Sequel ORM](http://sequel.jeremyevans.net) instead of ActiveRecord. The
+important things to note here are the `:tactical_eager_loading` plugin, which
+will greatly reduce the number of queries performed when sideloading related
+records, and the `:skip_collection_check` option that must be set to true in
+order for `JSONAPI::Serializer.serialize` to be able to serialize single
+Sequel::Model instances.
+
+```ruby
+require 'sequel'
+require 'sinatra/base'
+require 'json/ext'
+require 'jsonapi-serializers'
+
+class Post < Sequel::Model
+  plugin :tactical_eager_loading # N.B.
```

Comment:
Can you drop this `# N.B.` and the one below? I think it's kind of an obscure reference most people won't get.